changelog.d/2928.removal.rst

```diff
@@ -0,0 +1 @@
+The ``set owner`` and ``set token`` commands have been removed in favor of managing server side. 
```

Comment:
Since we can call Discord's guild "servers" too, I found this changelog entry a little confusing when I first read it. Perhaps you could change it to "in favor of managing it through command line"?
```suggestion
The ``set owner`` and ``set token`` commands have been removed in favor of managing it through command line. 
```

Reply:
I'm electing to leave it as-is for now. It is technically possible to change later if needed, but this is clear enough for this particular case.